lymph/unilateral.py

```diff
@@ -148,6 +150,19 @@ def init_edges(self, graph):
                     self.lnl_edges.append(new_edge)
 
 
+    def init_parameters(self):
+        """Initializes the paramaters dictionary including all edges.
+        """
+        self.parameters = {}
+        for edge in self.tumor_edges:
+            self.parameters['tumor' + '_to_' + edge.end.name] = edge #we could rename this to spread as well.

```

Comment:
By the way, instead of computing the name here, we could/should implement a `name` property in the `Edge` class and simply call `edge.name`. You know, separation of concerns and all 😉

Reply:
Yes, I thought so too. Will be moving the naming to the edge class 👍🏽